### DISCUSSION TITLE
Stable Version

### DISCUSSION THREAD

**Discussion by je-almeida** [vercel/next.js#52043] [Help] [answered]
### Summary

Hello. How do I find the stable version?
Excuse my ignorance, but I don't know how to identify.
There are many recent versions marked as "Pre-release" and I'm looking for a stable and supported version.

If I install according to the documentation: npx create-next-app@latest, I imagine I would be using a beta version.

**Comment by icyJoseph** [comment, marked as answer]
Next.js works with canaries and stable releases. The latest release is https://github.com/vercel/next.js/releases/tag/v13.4.7 

> next@canary is the canary version of Next.js that ships daily. It includes all features and fixes that have not been released to the stable version yet. Think of canary as a public beta.

Typically running create-next-app checks and uses the latest stable, it'll ask you to install a new stable if it finds one.

So to use canary really, you need to do something like `yarn add next@canary` or npm equivalent, or you'd need to install first create-next-app@canary

So the stable version is what you get by installing next, next@latest, or rolling a new app with create-next-app - canaries require explicit opt-in.
